k8s-subst

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# This script generates K8s yaml files from a set of templates.  To run it you need to 
+# define env vars IMAGE_PATH and IMAGE_TAG such that clearwater images can be found at
+#   IMAGE_PATH/{container name e.g. bono}:IMAGE_TAG
+# The script takes a single parameter which is the path to the Kubernetes directory 
+# (which has the templates subdirectory).
+
+[ -z "$IMAGE_PATH" ] && { echo "Need to set IMAGE_PATH"; exit 1; }
+[ -z "$IMAGE_TAG" ] && { echo "Need to set IMAGE_TAG"; exit 1; }
+[ -z "$1" ] && { echo "Need to specify path to K8s tmpl files"; exit 1; }
```

Comment:
It's actually the path to the directory with the templates dir in it, right? If i passed in kubernbetes/templates/ this would fail, despite that being the path of the tmpl files.
Message should be clear what it's requiring, and possibly include an example. 
Equally, do we actually even need this? Can't the script just work relative to its location and 'know' where the templates are without user input. 
It would mean you can't run it over different templates as easily, but i don't think we need/want that flexibility. Should just be a 'run this script here' thing